content/docs/atom-guide/look-dev/vegetation-bending-tutorial.md

```diff
@@ -0,0 +1,716 @@
+4. Make sure the **Editor** is open, if it is not already open.
+5. Save your file with **CTRL-S** and the **Asset Processor** should automatically detect changes and process the file. You can open the **Asset Processor** and check when the file is done processing. 
+   {{< note >}}
+   If you can't find the **Asset Processor**, click the arrow on the bottom right of your taskbar, and then click on the icon of two arrows in a circle.
+   {{< /note >}}
+6. Once the **Asset Processor** is done processing the changes, you should see in the **Editor** that your material looks different!
+
+{{< image-width src="/images/atom-guide/vegetation-bending-tutorial/offset.png" width="100%" alt="Forward pass offset." >}}
+
+The main texture of our shader ball seems to be showing up at an offset as we intended, but a grey outline is still at the origin of the object. This is because we only edited the forward pass, but have yet to edit the *depth pass*. All the passes we go through are referenced in `VegetationBending.materialtype`.
+More information about passes can be found [here](../dev-guide/passes/), but keep in mind that different passes render different parts of the material, and some passes' outputs are used as inputs to other passes.
+
+Let's repeat the above steps but with the depth pass:
+1. Open `{your-project-path}\Materials\Types\VegetationBending_DepthPass.azsli`.
+   - Make sure you are not editing the `VegetationBending_DepthPass_WithPS.azsli` file.
+2. Find the function `DepthPassVS`.
+3. Towards the end of the function, right before `OUT.m_position = mul(ViewSrg::m_viewProjectionMatrix, worldPosition);`, add: 
+   
+   ```
+   worldPosition.x += 5;
+   ```
+4. Save your file and check our the **Editor**. The shader ball should now be completely rendered at an offset! 
+   - Note that the shadow is still in the original position. After finishing this tutorial, see if you can adjust the shadow yourself!
+
+{{< image-width src="/images/atom-guide/vegetation-bending-tutorial/fulloffset.png" width="100%" alt="Completed offset added." >}}
+
+### Add material property parameters
+For now, we have just been moving the ball an offset of `5` units. However, we may want an easy way to change the offset in the **Editor** instead of having to change the code. We can do this with adjustable parameters in the **Material Editor**.
+
+1. Open `{your-project-path}\Materials\Types\MaterialInputs\VegetationBendingPropertyGroup.json`.
+2. Under `properties`, you'll see that there is already a property, `xOffset`, written there for you. Following `xOffset` as a guide, add another property `yOffset`. The code should end up looking something like this:
```

Comment:
I like the way that this step gives them a working example with xOffset, but also lets them do it themselves for yOffset, instead of all or none. :)

Reply:
Glad to hear!